docs/data/charts/tooltip/tooltip.md

```diff
@@ -55,6 +55,21 @@ This kind of interaction is controlled by series properties `highlightScope` whi
 
 {{"demo": "ElementHighlights.js"}}
 
+### Controlled Highlight
```

Comment:
@alexfauquette should we move highlights to a dedicated page?

Reply:
Yes this page start to grow, but can be done in a distinct PR